ftml/conf/modules.toml

```diff
@@ -0,0 +1,28 @@
+[Backlinks]
+body = "none"
+[Backlinks.arguments]
+page = { type = "string" }
+
+[Categories]
+body = "none"
+[Categories.arguments]
+includeHidden = { type = "bool", default = false }
+
+[CSS]
+body = "raw"
+
+[Join]
+body = "none"
+html-attributes = true
+[Join.arguments]
+button = { type = "string", default = "" } # default message depends on language
+
+[PageTree]
+body = "none"
+[PageTree.arguments]
+root = { type = "string" }
+depth = { type = "int", min-value = 1, default = 10000 } # really unlimited, but not sure how to convey that as an int
```

Comment:
-1?

Reply:
I could see the -1 having issues? While it's okay here because the minimum value is above 0, if it wasn't above 0 it would be a valid default. I think having a weird check for that is kind of meh. Maybe just have a special "inf" value or something? Or just leave it like this, I'm not sure how big a deal this is.